atd-vze/README.md

```diff
@@ -1,3 +1,25 @@
+# Vision Zero Editor (VZE)
```

Comment:
📝

Reply:
yeah, I need to get into a better habit of documenting more.

Reply:
Me too!